iothub/service/src/Twin/TwinsClient.cs

```diff
@@ -169,7 +175,7 @@ public virtual async Task<Twin> GetAsync(string deviceId, string moduleId, Cance
         /// certificate validation.
         /// </exception>
         /// <exception cref="OperationCanceledException">If the provided <paramref name="cancellationToken"/> has requested cancellation.</exception>
-        public virtual async Task<Twin> UpdateAsync(string deviceId, Twin twinPatch, string etag, CancellationToken cancellationToken = default)
+        public virtual async Task<Twin> UpdateAsync(string deviceId, Twin twinPatch, ETag etag, bool onlyIfUnchanged = false, CancellationToken cancellationToken = default)
```

Comment:
The ETag for this twin is already in the ```twinPatch``` argument, so we don't need to ask for it as a separate parameter here

Reply:
```suggestion
        public virtual async Task<Twin> UpdateAsync(string deviceId, Twin twinPatch, bool onlyIfUnchanged = false, CancellationToken cancellationToken = default)
```